app/src/main/res/layout/card_support.xml

```diff
@@ -1,61 +1,65 @@
-<android.support.v7.widget.CardView
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<android.support.design.card.MaterialCardView xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     style="@style/StandardCard">
 
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:background="@drawable/card_selector"
         android:orientation="vertical"
         android:padding="@dimen/material_default_padding">
 
         <TextView
+            style="@style/CardTitle"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:text="@string/support_us_title"
-            android:textAppearance="?android:attr/textAppearanceMedium"
-            android:textSize="20sp"
-            android:gravity="center_horizontal"
-            android:typeface="sans"/>
+            android:text="@string/support_us_title" />
 
         <TextView
+            style="@style/CardBody"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_marginBottom="@dimen/material_default_padding"
-            android:layout_marginTop="@dimen/material_small_padding"
-            android:text="@string/support_us"/>
+            android:text="@string/support_us" />
 
-        <android.support.v7.widget.AppCompatButton
+        <android.support.design.button.MaterialButton
```

Comment:
Having 3 buttons on top of each other in that support card actually looks really ugly.
I think we need to cut down on text on those buttons.
I'd suggest: "Like", "Fork", "E-mail", since we actually explained those buttons in the card's text already

Reply:
Agreed. I had yet to think of a better solution. Will try yours.